README.md

```diff
@@ -8,57 +8,59 @@ provided "as is"; we make no guarantees about any plugin in this repo.
 
 ## Creating new plugins
 There are two methods to create an external plugin, you can either:
+
  - Use [this](https://github.com/runelite/example-plugin/) plugin template.
 
  - Clone this repository and run the `create_new_plugin.py` script. This requires you to have `python3` installed
 
+**If you are using IntelliJ, you will need at least version 2017.3**
```

Comment:
Is this really relevant in 2020?

Reply:
Somebody had trouble with it, so it thought there was no harm in adding the information.